CHANGELOG-AUTO.md

```diff
@@ -0,0 +1,203 @@
+
```

Comment:
We already have a CHANGELOG.md in root folder. I don't think it is a good idea to have 2 changelog files. Maybe I am misunderstanding the strategy behind this new file.

Reply:
So the old CHANGELOG was a manually created file. That dose not really make sense to maintain I had wanted to see how well the auto generated changelog tooling worked for a while before removing and/or deprecating the old CHANGELOG.md. If you think we should just do that now we can. One issue though is that the auto generated tooling replaces the whole file each time, so if we wanted to keep the old one we would need to rename it something like CHANGELOG-v0.1.0-v1.1.0.md or something like that and then just have the new auto generated stuff go into CHANGELOG.md

Reply:
I will also point out slightly this first run for 1.4 is also not 100% perfect because sematic pr titles where note enabled for all of it so there is a few wrong headings etc but going forward it should be reallly good.

Reply:
Renaming the old one makes sense to me. I agree that the new file should be just CHANGELOG.md.

Reply:
Sounds good let me update